src/main/java/study/Numbers.java

```diff
@@ -0,0 +1,26 @@
+package study;
+
+import java.util.ArrayList;
+
+public class Numbers {
+    private static final String MATCH_NUMBER = "^[0-9]+$";
+    private static final String EXCEPTION_MESSAGE = "음수 또는 정수 이외의 값은 더할 수 없습니다.";
+
+    private ArrayList<Integer> numbers = new ArrayList<>();
+
+    public  Numbers(String[] numberList) {
+        for (int i = 0; i < numberList.length; i++) {
+            validateNumberList(numberList[i]);
+            this.numbers.add(Integer.parseInt(numberList[i]));
+        }
+    }
+
+    public void validateNumberList(String number) {
+        if (!number.matches(MATCH_NUMBER)) {
+            throw new RuntimeException(EXCEPTION_MESSAGE);
+        }
+    }
+    public ArrayList<Integer> getNumbers() {
```

Comment:
내부에서 사용하는 메소드는 `private` 접근 제어자로 `캡슐화`해보면 좋겠네요.

```suggestion
    public void validateNumberList(String number) {
        if (!number.matches(MATCH_NUMBER)) {
            throw new RuntimeException(EXCEPTION_MESSAGE);
        }
    }
    
    public ArrayList<Integer> getNumbers() {
```